test/new-e2e/system-probe/test-runner/main.go

```diff
@@ -106,7 +113,7 @@ func pathToPackage(path string) string {
 
 func buildCommandArgs(pkg string, xmlpath string, jsonpath string, file string, testConfig *testConfig) []string {
 	args := []string{
-		"--format", "dots",
+		"--format", "testname",
```

Comment:
This will significantly increase the size of the log output. The idea behind `dots` was the ensure the failures were easy to spot.

Reply:
Failures are highlighted in the bottom, so it should still be easy to spot them.
Secondly, since tests are distributed across various jobs, having the testname in the log output would be a convenient way to be certain that a job is responsible for the test a user is interested in.